### DISCUSSION TITLE
Mail goes to the queue instead of being sent

### DISCUSSION THREAD

**Discussion by bjoworld** [symfony/symfony#45667] [Q&A]
Hello

Until now I never had problems sending mails in Symfony projects. For a new project I used the new 6 version. I send the mails out via SMTP, however this always goes into the queue instead of being sent.
Does anyone have an idea what this could be?

Thank you very much.

Best regards
Björn

![Bildschirmfoto 2022-03-07 um 18 27 50](https://user-images.githubusercontent.com/48091863/157086232-51c6eec9-6aab-4f06-a0e5-f153f39fe356.png)

**Comment by mbabker** [comment]
If `symfony/messenger` is installed and configured, emails will be queued through Messenger instead of immediately sent.  Whether Messenger is set to send immediately or asynchronously through the `messenger:consume` command is dependent on your site config (check the `config/packages/messenger.yaml` file to see how things are set up).

**Reply by FrenchKoyote** [reply]
@mbabker you are the best ;)
